src/servers/main.cc

```diff
@@ -245,25 +245,25 @@ CheckPortCollision()
 {
   // Check if HTTP and GRPC have shared ports
   if ((std::find(http_ports_.begin(), http_ports_.end(), grpc_port_) !=
-       http_ports_.end()) && (grpc_port_ != -1) &&
-       allow_http_ && allow_grpc_) {
+       http_ports_.end()) &&
```

Comment:
These changes seem unrelated to the PR and should go in a different PR

Reply:
Automatic changes due to formatting + rebase. I can remove these as these changes are already in master